docs/build/locally.adoc

```diff
@@ -0,0 +1,146 @@
+=== Building Locally
+
+All of the Docker images used for {project_name} can be built using he Independent Runner. However some tools need to be installed locally as the first two images must be built on the local machine. The other images can then be built using EIR as they are built using the initial images.
+
+The tools required for building the initial images are as follows:
+
+* Docker Engine
+** Docker Server
+** Docker Desktop - https://www.docker.com/products/docker-desktop/
+** Rancher Desktop - https://rancherdesktop.io 
+** POD Man - https://podman.io
```

Comment:
Dunnno if this is just PR rendering issues but the servers aren't under the Docker Engine here...

Reply:
When this is rendered in AsciiDoc and then transformed it looks ok.
Not sure I understand what you mean here.

Reply:
If you click the render button on the PR it's all on the same level, Maybe it works once it's all stitched together